### ISSUE TITLE
[.NET 8] Warning: "Found version-specific or distribution-specific runtime identifiers"

### ISSUE THREAD

**Issue by ladenedge** [Azure-Samples/cognitive-services-speech-sdk#2244]
### **Describe the bug**

Including the [Microsoft.CognitiveServices.Speech NuGet package](https://www.nuget.org/packages/Microsoft.CognitiveServices.Speech) yields a build warning.

```
C:\Program Files\dotnet\sdk\8.0.100\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.targets(284,5):
   warning NETSDK1206: Found version-specific or distribution-specific runtime identifier(s): centos7-x64.
   Affected libraries: Microsoft.CognitiveServices.Speech. In .NET 8.0 and higher, assets for version-specific
   and distribution-specific runtime identifiers will not be found by default.
```

Note this is similar (but not identical) to #2170.  As I mentioned over there, instead of listing many RIDs, I am now only seeing **centos7-x64**.

### **To Reproduce**

1. Create an empty .NET 8 Console App.
2. Install the Microsoft.CognitiveServices.Speech NuGet package.
3. Build the solution.

### **Expected behavior**

No warnings!

### **Version of the Cognitive Services Speech SDK**

Microsoft.CognitiveServices.Speech **v1.34.1**.

### **Platform, Operating System, and Programming Language**

 - OS: Windows
 - Hardware: x64
 - Programming language: C#
 - Visual Studio 2022, .NET 8

Thanks for your time!

**Comment by ladenedge** [comment]
Still present in 1.35.0.

**Comment by pankopon** [comment]
This appears due to RHEL/CentOS 7 specific binaries under `runtimes\centos7-x64` that we have to keep around at least until the [CentOS 7 EOL](https://www.redhat.com/en/topics/linux/centos-linux-eol) or so. As documented for this [breaking change in .NET 8](https://learn.microsoft.com/dotnet/core/compatibility/sdk/8.0/rid-graph), one workaround is to set `UseRidGraph` true in the project file.

**Comment by ladenedge** [comment]
Okay, thank you for the update!

**Comment by exedor** [comment]
I'm running into the same thing with a fairly large project with many thousands of lines of code. It's the only warning I have in the project :) Are there any side-effects to the above mentioned work around?

**Comment by pankopon** [comment]
@exedor The linked .NET 8 documentation does mention that
>  The long-term goal is to stop updating the RID graph, stop reading it, and eventually remove it.

but the workaround should be good at least for the .NET 8 lifetime, and hopefully by the end of it we will have removed the binaries that trigger the warning.

**Comment by pankopon** [comment]
Internal work item ref. 6718573. A possible target is soon after the CentOS 7 EOL on June 30.

**Comment by pankopon** [comment]
This has been fixed in the Speech SDK 1.40.0 release out now.
